src/DipStrikeTool.cpp

```diff
@@ -100,28 +100,26 @@ DipStrikeTool::DipStrikeTool(std::shared_ptr<cs::core::InputManager> const& pInp
     std::string const& sFrame)
     : MultiPointTool(pInputManager, pSolarSystem, graphicsEngine, pTimeControl, sCenter, sFrame)
     , mGuiArea(new cs::gui::WorldSpaceGuiArea(420, 225))
-    , mGuiItem(new cs::gui::GuiItem("file://../share/resources/gui/dipstrike.html"))
-    , mVAO(new VistaVertexArrayObject())
-    , mVBO(new VistaBufferObject())
-    , mShader(new VistaGLSLShader()) {
+    , mGuiItem(new cs::gui::GuiItem("file://../share/resources/gui/dipstrike.html")) {
```

Comment:
I can't leave a suggestion here, because GitHub complains about deleted lines, but we should use `std::make_unique` in both cases.